ui/views.py

```diff
@@ -57,7 +57,8 @@ def get(self, request, *args, **kwargs):
             "name": name,
             "username": username,
             "host": webpack_dev_server_host(request),
-            "edx_base_url": settings.EDXORG_BASE_URL
+            "edx_base_url": settings.EDXORG_BASE_URL,
+            "search_url": "http://localhost:9200",
```

Comment:
so i think OSX users sort of have 2 different Elasticsearch URLs. The value of `ELASTICSEARCH_URL` in docker-compose.yml is the one is used in our Python code (which obviously runs inside the container). the other URL would be for communication from outside the container. i think we might want to create a new .env value for this. `ELASTICSEARCH_EXTERNAL_URL` or something like that? 

i was thinking we might want to move `ELASTICSEARCH_URL` out of docker-compose.yml and into .env, but that current `ELASTICSEARCH_URL` is still relevant for communication